### ISSUE TITLE
A plain enum defined in C# is returned as a generic FsharpSymbol

### ISSUE THREAD

**Issue by 7sharp9** [fsharp/FSharp.Compiler.Service#410]
```
System.StringSplitOptions.RemoveEmptyEntries
```

This is returned as a FSharpSymbol which means that the properties of the underlying type are not available.  i.e. The field literal value is not exposed to no tooltips can be generated against this type.  i.e.
the filed type and literal value should be exposed via a `FsharpField` symbol type.

![screen shot 2015-08-28 at 15 34 03](https://cloud.githubusercontent.com/assets/588746/9548765/4953e59a-4d9a-11e5-9a17-7a82ab11515b.png)

As it stands this generic symbol is no use for consuming tools.

**Comment by 7sharp9** [comment]
@dsyme What would be the logical symbol type?  `FSharpField`?

The underlying type is `Item.ILField`

**Comment by 7sharp9** [comment]
I don't think this is correct but some help to determine what will be correct would be appreciated.
```fsharp
        | Item.ILField(ILFieldInfo(iLTypeInfo, iLFieldDef)) -> 
             let f = mkRecdFieldRef iLTypeInfo.TyconRef iLTypeInfo.Name
             FSharpField(cenv,  f) :> _ 
```

Im thinking that the FSharpFieldData would be sufficient to represent the ILFiled which is after all a field.

```fsharp
and FSharpFieldData = 
    | Recd of RecdFieldRef
    | Union of UnionCaseRef * int
    member x.RecdField =
        match x with 
        | Recd v -> v.RecdField
        | Union (v,n) -> v.FieldByIndex(n)
    member x.DeclaringTyconRef =
        match x with 
        | Recd v -> v.TyconRef
        | Union (v,_) -> v.TyconRef 
```

**Comment by dsyme** [comment]
Fixed by #436
